CHANGELOG.md

```diff
@@ -1777,4 +1788,6 @@ That's why is this section formatted differently.
 [#503]: https://github.com/shopsys/shopsys/pull/503
 [v7.0.0-beta2]: https://github.com/shopsys/shopsys/compare/v7.0.0-beta2...v7.0.0-beta2
```

Comment:
hmm, this is strange

Reply:
It links the phrase `v7.0.0-beta2` that is not even there :slightly_smiling_face: It should not be there so I removed it.